test_runner/fixtures/neon_fixtures.py

```diff
@@ -2645,6 +2649,43 @@ def layer_exists(
         layers = self.list_layers(tenant_id, timeline_id)
         return layer_name in [parse_layer_file_name(p.name) for p in layers]
 
+    def timeline_scan_no_disposable_keys(
```

Comment:
This function would benefit from a docstring ("disposable keys" is a bit of an obscure concept when one isn't actively working on sharding stuff)